cardano-node/src/Cardano/Node/Tracing/Tracers/Startup.hs

```diff
@@ -141,6 +143,24 @@ namesStartupInfo = \case
   BIByron {}                                -> ["Byron"]
   BINetwork {}                              -> ["Network"]
 
+
+data VersionTuple a b = VersionTuple a b
```

Comment:
Why bother with introducing an additional datatype when we can just use `(,)`?

Reply:
Because then we can control the `ToJSON` instance.

Reply:
Fair enough, can we rename this to `NodeVersionTuple`?

Reply:
Sure